infra/ecs/orcavault-dbt/main.tf

```diff
@@ -66,5 +66,9 @@ data "aws_rds_cluster" "orcahouse_db" {
 }
 
 data "aws_ssm_parameter" "ro_username" {
-  name = "/${local.stack_name}/ro_username"
+  # For daily scheduled dbt run; we have to run post-ELT hook grant select
+  # to Athena db ro user for mart schema. We pass this ro username via ECS
+  # task definition env var at terraform deploy time.
+  # See orcavault/macros/grant_select.sql
+  name = "/${local.stack_name}/${local.database_name}/athena_username"
```

Comment:
You still want to construct the SSM parameter name from stack variables? 
The parameter is set up manually and completely independent of this stack. I'd just use it as a given constant. That way it's guarded against changes within the stack. 

No big deal though, as this is unlikely to ever change or cause issues.

Reply:
Good idea. I will update it.